php/utils.php

```diff
@@ -215,7 +215,7 @@ function handle_exception($exception) {
  * @return [boolean] $is_hermes True if Terminus is operating on Hermes
  */
 function is_hermes() {
-  $is_hermes = (TERMINUS_HOST == 'dashboard.getpantheon.com');
```

Comment:
I think we can remove this entire function. The only place I see it being referenced is in this file, and in a unit test for this file. (though please verify).